python-avd/pyavd/_eos_designs/schema/schema_fragments/ipsec_settings.schema.yml

```
@@ -0,0 +1,22 @@
+# Copyright (c) 2023-2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ipsec_settings:
+    documentation_options:
+      table: ipsec-settings
```

For now let's keep it with other wan settings. The generated tables will now show up in the docs unless you add an include statement in the "input-variables.md" file.

```suggestion
    documentation_options:
      table: wan-settings
```